src/app/pages/HomePage/index.tsx

```diff
@@ -25,94 +24,6 @@ export const zIndexHomePage = {
   mobileTooltip: 4,
 }
 
-const HomepageLayout = styled(Box)(({ theme }) => ({
```

Comment:
we can omit migration of HomePage and its components in next PRs as we gonna nuke that page and replace with completely new layout.